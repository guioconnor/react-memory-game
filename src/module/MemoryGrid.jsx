import React from 'react';
import styled from 'styled-components'

const Grid = styled.div`
  display: flex;
  flex-wrap: wrap;
  width: 80vmin;
  height: 80vmin;
  margin: 0 auto 0 auto;
  user-select: none;
  background: green;

  & a {
    width: auto;
  }
`;

export default (({ children }) => <Grid>{children}</Grid>);