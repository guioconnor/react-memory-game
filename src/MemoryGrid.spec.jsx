import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import MemoryGrid from './MemoryGrid';

describe('<MemoryGrid>', () => {
  const MockChildren = () => <div>Mock Children</div>;
  const container = shallow(<MemoryGrid>{MockChildren}</MemoryGrid>);

  it('should render a container with children', () => {
    expect(toJson(container)).toMatchSnapshot('MemoryGrid');
  });
});
