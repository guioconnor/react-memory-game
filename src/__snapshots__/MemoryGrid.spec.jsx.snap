// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MemoryGrid> should render a container with children: MemoryGrid 1`] = `
<div
  className="memory-game--memory-grid"
/>
`;
